Create Guión para la presentación del proyecto.

LuisGilDzib opened the pull request:
Se adjunta el guion de prueba, realizé una prueba de tiempo, el cual fue de 4:19 minutos, así que si se desea agregar más información, esta deberá ser muy corta

LuisGilDzib committed:
Create Guión para la presentación del proyecto.

guión de prueba

ricardo-andre-pool-cen commented on Guión para la presentación del proyecto.:
```diff
@@ -0,0 +1,38 @@
+Al momento de definir los requisitos hubo muchas dudas, que a lo largo del curso se fueron puliendo, hasta llegar a las actuales, estos requisitos nos mencionan una idea 
+generalizada del prototipo no funcional, como la línea del tiempo, o las asignaciones, roles, ramas, entre otras descripciones del repositorio.
+- Priorización
+En la parte de priorización se muestra el análisis de nuestros requisitos mostrandolo para cada uno, su importancia y factibilidad, mostrando el método usado
```
a la parte de priorizacion se refiere a filtrar requisitos que debemos tener, podriamos tener y no vamos a tener, para aligerar la carga

ricardo-andre-pool-cen commented on Guión para la presentación del proyecto.:
```diff
@@ -0,0 +1,38 @@
+- Artefactos.
+Al igual que hemos incluido artefactos que especifican los requisitos de nuestro proyecto, así que si quieres conocer más a detalle nuestro proyecto, puedes visitar nuestro
+repositorio, aquí en la descripción de nuestro video se adjunto el link de nuestro repsoitorio, conocerás nuestro modelo de trabajo, la descripción detallada y funcionalidad
+de nuestro proyecto, así que anímate a visitarnos y conocer más sobre nuestro grandioso seudorepositorio.
```
No creo que sea necesaria una despedida tan esplendida

ricardo-andre-pool-cen reviewed (changes requested):
En si, esta bien, el contenido que no se describa lo puedo mostrar en la edición, solo aplica los cambios que te dije

LuisGilDzib commented:
Ok

LuisGilDzib committed:
Update Guión para la presentación del proyecto.

he actualizado y aplicado los cambios solicitados, es espera de una nueva revisión.

LuisGilDzib commented:
Cambios aplicados, en espera de revisión.

ricardo-andre-pool-cen reviewed (approved):
Ese será ser nuestro guion
